chapters/annotations.tex

```diff
@@ -1364,64 +1383,91 @@ \subsubsection{Edit Input}\label{edit-input}
 
 \section{Graphical User Interface}\label{annotations-for-the-graphical-user-interface}\label{graphical-user-interface}
 
-This section describes the annotations that are used to define properties of the graphical user interface.
+The annotations listed below define properties for use in graphical user interfaces.
+\begin{center}
+\begin{tabular}{l|l l}
+\hline
+\tablehead{Annotation} & \tablehead{Description} & \tablehead{Details}\\
+\hline
+\hline
+{\lstinline!preferredView!} & Default view when opening class & \Cref{modelica:preferredView}\\
+{\lstinline!DocumentationClass!} & Purpose of class is documentation & \Cref{modelica:DocumentationClass}\\
+{\lstinline!defaultComponentName!} & Default name for new components & \Cref{modelica:defaultComponentName}\\
+{\lstinline!defaultComponentPrefixes!} & Default type prefixes for new components & \Cref{modelica:defaultComponentPrefixes}\\
+{\lstinline!missingInnerMessage!} & Message for unresolved \lstinline!outer! & \Cref{modelica:missingInnerMessage}\\
+{\lstinline!absoluteValue!} & Quantity is absolute & \Cref{modelica:absoluteValue}\\
+{\lstinline!defaultConnectionStructurallyInconsistent!} & Relax verification of non-simulation class & \Cref{modelica:defaultConnectionStructurallyInconsistent}\\
```

Comment:
```suggestion
{\lstinline!defaultConnectionStructurallyInconsistent!} & Relax verification of connections & \Cref{modelica:defaultConnectionStructurallyInconsistent}\\
```
I think that mentioning non-simulation is more confusing than helping, but I'm not sure if the new text is good.

Reply:
How about this?
```suggestion
{\lstinline!defaultConnectionStructurallyInconsistent!} & Suppress certain verification errors & \Cref{modelica:defaultConnectionStructurallyInconsistent}\\
```

Reply:
Ok, looks ok

Reply:
Or, "balancing" instead of "verification"? (The problem is how to describe so that it makes sense - it doesn't mean that the models are less correct overall; just different.)

Reply:
Yes, that's one of the advantages with _certain verification errors_; it has less of the spirit of being full-blown errors.